contrib/http_util/src/test/java/io/opencensus/contrib/http/FakeSpan.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright 2018, OpenCensus Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.opencensus.contrib.http;
+
+import io.opencensus.trace.Annotation;
+import io.opencensus.trace.AttributeValue;
+import io.opencensus.trace.EndSpanOptions;
+import io.opencensus.trace.Link;
+import io.opencensus.trace.MessageEvent;
+import io.opencensus.trace.Span;
+import io.opencensus.trace.SpanContext;
+import io.opencensus.trace.Status;
+import java.util.Map;
+
+/** A fake {@link Span} which allows user to set {@link SpanContext} upon initialization. */
+public class FakeSpan extends Span {
```

Comment:
Please add this in the opencensus-testing package. See https://github.com/census-instrumentation/opencensus-java/pull/410 . I probably need to merge that.

Reply:
For the moment mark this as internal so that we can remove it later. I don't want you to get block by my comment. Also if possible to make it package protected that is even better.